core/shared/src/main/scala/org/http4s/ContentCoding.scala

```diff
@@ -15,7 +15,7 @@
  */
 
 /*
- * Copyright 2013-2020 http4s.org
+ * Copyright 2013-2022 http4s.org
```

Comment:
Spurious change? :) see also https://github.com/http4s/http4s/pull/5525

Reply:
Ahhh, thanks! It's good to know. Will rectify :-)